server/hooks/after_copyCartToOrder.js

```diff
@@ -1,63 +1,72 @@
-function shipmentChecker(date) {
-  if (moment(date).isoWeekday() === 7) {
-    return moment(date).subtract(2, 'days')._d;
-  } else if (moment(date).isoWeekday() === 6) {
-    return moment(date).subtract(1, 'days')._d;
-  }
-  return date;
-}
+// function shipmentChecker(date) {
+//   if (moment(date).isoWeekday() === 7) {
+//     return moment(date).subtract(2, 'days')._d;
+//   } else if (moment(date).isoWeekday() === 6) {
+//     return moment(date).subtract(1, 'days')._d;
+//   }
+//   return date;
+// }
 
```

Comment:
Lots of commented out functions here as well. What would make you comfortable deleting?